.pre-commit-config.yaml

```diff
@@ -4,13 +4,18 @@ repos:
     hooks:
       - id: ruff
       - id: ruff-format
```

Comment:
unrelated but wondering: should we add "ruff check --fix" too for ordering imports?